### DISCUSSION TITLE
当使用了简悦自带的字体或自定义字体后无法生效的说明

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1864] [重要通知]
## 问题描述

RT

## 问题原因

1. 确保 **这些字体已经安装在你的 PC**
2. 确保 Chrome （或扩展）没有更改过全局字体 _某些 Chromium 变种浏览器有这种选项_

另外，确实在 Windows 端偶尔存在字体无法更改的情况。（大多数问题都因为 **第二点** 造成）

## 排查方案

1. 隐私模式下，仅启用简悦，来排查其它扩展造成的干扰。
2. 导出配置文件后，重新安装简悦，并不改变任何配置的前提下测试。

如果以上仍无法解决你的问题，则说明你的浏览器设置了某些全局类的字体样式等，所以请使用一个干净的浏览器测试。

## 关联描述

[利用 Google Fonts 为阅读模式添加任何字体](https://github.com/Kenshin/simpread/discussions/1849)

**Comment by kiyoakii** [comment]
我在使用简阅前，更改过 Chrome 的全局字体设置，在使用简阅并修改字体无效后，我尝试恢复 Chrome 默认设置，但发现字体仍然没有恢复成功。我是付费用户，请问有什么解决方式吗？（不考虑重装 Chrome，恢复默认设置的成本已经足够高了）

**Comment by kiyoakii** [comment]
接上条，已经解决了。
macOS 用户可以在关闭 Chrome 后，修改 ~/Library/Application Support/Google/Chrome/Default/Preferences 文件，将
```
{
"default_fixed_font_size":13,
"default_font_size":16,
"fonts":{
  "fixed":{"Zyyy":"<self-font>"}, 
  "sansserif":{"Zyyy":"<self-font>"},
  "serif":{"Zyyy":"<self-font>"},
  "standard":{"Zyyy":"<self-font>"}
}
```
替换为
```
{
"default_fixed_font_size":13,
"default_font_size":16,
"fonts":{
  "fixed":{"Zyyy":""},
  "sansserif":{"Zyyy":""},
  "serif":{"Zyyy":""},
  "standard":{"Zyyy":""}
}
```

**Reply by Kenshin** [reply]
嗨，你太能「折腾」了，我以为是 Win，没想到 Mac 也可以这么魔改字体...

搞定就好，我都不知道这种操作。😂

PS：修改字体的功能是基础账户就有的，跟高级账户无关。

**Reply by tinggengz** [reply]
win10 环境下是，C:\Users\“自己用户名称”\AppData\Local\Microsoft\Edge\User Data\Default\Preferences  这个文件可以找到。
然后跟 @kiyoakii 提到的一样的修改，这个步骤是重置掉edge浏览器的默认字体设置，主要是可视化操作没能选，只能该配置文件了。

其实我改完后，本来以为可以根 楼上一样的效果，后面发现还跟一些 修改浏览器样式的插件有关，比如 "Dark Reader" 这个插件，因为之前自己"Dark Reader"  时已经启用并置了默认字体了，造成了影响。不过改完后就可以了。

**Reply by Kenshin** [reply]
@tinggengz 

嗨，谢谢把操作过程放上来「以待后来者」🙏

**Comment by erichpcs** [comment]
苹果字体是我的最爱，其次是楷体。

**Reply by Kenshin** [reply]
哈哈，简悦也支持多种定制化字体方式，比如 霞鹜文楷 等，细节可以 [看这里](https://github.com/Kenshin/simpread/discussions?discussions_q=label%3Atheme)。
